internal/discovery/.gitignore

```diff
@@ -0,0 +1,4 @@
+/organization_list.json
+/organization_list.json.tmp.minisig
```

Comment:
Any reason why these contain `.tmp`?

Reply:
Actually, I removed this completely from this PR, as this `.gitignore` is entirely specific for my build system.